libs/sql-parser/src/main/antlr/SqlBase.g4

```diff
@@ -408,7 +410,6 @@ unquotedIdent
     : IDENTIFIER                        #unquotedIdentifier
     | nonReserved                       #unquotedIdentifier
     | DIGIT_IDENTIFIER                  #digitIdentifier        // not supported
-    | COLON_IDENT                       #colonIdentifier        // not supported
```

Comment:
The slices with whitespaces worked without this change `arr[1 : 2]`.
I had to remove the colon identifier because the colon, adjacent to numbers, was leading to an error. So this syntax didn't work `arr[1:2]`. I didn't find another way to make it work, other than removing the colon check.

Reply:
👍 was not supported anyway.